### PR TITLE
[New] Add jQuery-Paging@1.2.0 w/ git auto-update

### PR DESCRIPTION
**Pull request by sufuf3**
Pull request for issue: #10460
Related issue(s): #10547 #11170

Checklist for **Pull request** or **lib adding request issue** follows the conventions.

Note that if you are using a distribution purpose repository/package, please also provide the url and other related info like popularity of the source code repo/package.

# Profile of the lib
 * Git repository (required): https://github.com/infusion/jQuery-Paging
 * Official website (optional, not the repository): http://www.xarg.org/2011/09/jquery-pagination-revised/
 * NPM package url (optional):
 * License and its reference: MIT, GPL-2.0
 * GitHub / Bitbucket popularity (required):
   - Count of watchers: 12
   - Count of stars: 210
   - Count of forks: 82
 * NPM download stats (optional):
   - Downloads in the last day:
   - Downloads in the last week:
   - Downloads in the last month:

# Essential checklist
 * [ ] I'm the author of this library
   * [ ] I would like to add link to the page of this library on CDNJS on website / readme
 * [x] This lib was not found on cdnjs repo
 * [x] No already exist / duplicated issue and PR
 * [x] The lib has notable popularity
   * [x] More than 100 [Stars / Watchers / Forks] on [GitHub / Bitbucket]
   * [ ] More than 500 downloads stats per month on npm registry
 * [ ] Project has public repository on famous online hosting platform (or been hosted on npm)

# Auto-update checklist
 * [x] Has valid tags for each versions (for git auto-update)
 * [x] Auto-update setup
 * [x] Auto-update target/source is valid.
 * [x] Auto-update filemap is correct.

# Git commit checklist
 * [x] The first line of commit message is less then 50 chars, be clean and clear, easy to understand.
 * [x] The parent of the commit(s) in the PR is not old than 3 days.
 * [x] Pull request is sending from a non-master branch with meaningful name.
 * [x] Separate unrelated changes into different commits.
 * [x] Use rebase to squash/fixup dummy/unnecessary commits into only one commit.
 * [x] Close corresponding issue in commit message
 * [x] Mention related issue(s), people in commit message, comment.

### CODE REVIEW

**Comment by sufuf3** [comment]
@cdnjs/intern3 
Please help me review this PR, Thank you.

**Comment by sufuf3** [comment]
ping @clairetsai818 @hare1039

**Comment by sufuf3** [comment]
ping @hare1039 @clairetsai818

**Comment by sufuf3** [comment]
@cdnjs/library-reviewer 
Please help me review this PR, Thank you.

**Comment by sufuf3** [comment]
@hare1039 @qwe19272375 @pvnr0082t @maruilian11 
Please help me review this PR, Thank you.

**Comment by sufuf3** [comment]
@hare1039 @qwe19272375 @pvnr0082t @maruilian11
Please help me review this PR, Thank you.

File: ajax/libs/jQuery-Paging/package.json [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+  },
+  "keywords": [
+    "paging",
+    "pagination",
+    "pager",
+    "navigation"
+  ],
+  "author": {
+    "name": "Robert Eisele",
+    "url": "http://www.xarg.org/"
```
Comment (qwe19272375) [inline ajax/libs/jQuery-Paging/package.json]: maybe use `https`

**Comment by sufuf3** [comment]
@qwe19272375 I update the url 
Please help me review this PR, Thank you.

PS. This link had some ssl problem before, now I think it has fixed.

**Comment by sufuf3** [comment]
@maruilian11 @pvnr0082t 
Please help me review this PR, Thank you.

**Comment by sufuf3** [comment]
@PeterDaveHello 
Please help me review this PR, Thank you.

**Pull request merged**